const/meta.ts

```diff
@@ -8,7 +10,8 @@ export const siteConfig = {
     rootIPFS: "https://IPFS-URL-HERE",
     swap: "https://cowswap.exchange",
     docs: "https://docs.cowswap.exchange/",
```

Comment:
You could already update this URL which does exist (contrary to the API ones) to docs.cow.fi

Reply:
Good point, I tried to add all of them, i forgot this one